src/main/java/com/force/i18n/grammar/impl/SimpleDeclension.java

```diff
@@ -90,8 +90,9 @@ protected void setString(String value, NounForm form) {
         protected boolean validateValues(String name, LanguageCase _case) {
 ///CLOVER:OFF
             if (this.value == null) {
-                // this is wrong.  the noun should have singular form
-                logger.severe("###\tError: The noun " + name + " has no value");
+                // this is wrong. the noun should have singular form
+                HumanLanguage language = this.getDeclension().getLanguage();
+                logger.severe(() -> "###\tError: " + language.getLocaleString() + "The noun " + name + " has no value");
```

Comment:
show language information for SEVERE error.  also, refactor using `Logger#severe(Supplier<String>)` to construct log message lazily.